code/modules/hydroponics/grown/grass_carpet.dm

```diff
@@ -27,7 +27,7 @@
 	bitesize_mod = 2
 	var/stacktype = /obj/item/stack/tile/grass
 	var/tile_coefficient = 0.02 // 1/50
-	wine_power = 15
+	distill_reagent = "/datum/reagent/consumable/ethanol/beer/light"
```

Comment:
```suggestion
	distill_reagent = /datum/reagent/consumable/ethanol/beer/light
```
or
```suggestion
	distill_reagent = "light_beer"
```

Depending whether this will be merged before or after the reagents ID removal.